src/components/common/CardBlog/CardBlog.module.scss

```
@@ -0,0 +1,34 @@
+@import "../../../styles/utilities";
+
+.cardBlogWarpper{
+    max-width: 39.2rem;
+    min-height: 34.4rem;
+    @apply inline-flex flex-col justify-start;
+    .image{
+        width: 100%;
+        max-height: 22rem;
```

Your CSS code is not formated. Please check it

```suggestion
.cardBlogWarpper {
    max-width: 39.2rem;
    min-height: 34.4rem;
    @apply inline-flex flex-col justify-start;
    .image {
        width: 100%;
        max-height: 22rem;
```